modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -10,7 +10,7 @@ You can use the `FeatureGate` custom resource (CR) to enable specific feature se
 
 You can activate the following feature set by using the `FeatureGate` CR:
 
-* `TechPreviewNoUpgrade`. This feature set is a subset of the current Technology Preview features. This feature set allows you to enable these tech preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents minor version updates. This feature set is not recommended on production clusters.
+* `TechPreviewNoUpgrade`. This feature set is a subset of the current Technology Preview features. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents minor version updates. This feature set is not recommended on production clusters.
```

Comment:
Can we remove "Enabling this feature set cannot be undon.. " from here as it is mentioned in WARNING.

Reply:
Double checked with @skopacz1, who added the warning below, and he said it was fine to remove the duplicate text from this line.